codechefcli/decorators.py

```diff
@@ -33,3 +33,32 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
 
     return wrapper
+
+
+def sort_it(func):
+    '''
+    desc: decorator method to sort the specified argument
+    '''
+    def wrapper(*args, **kwargs):
+        sort = args[0]
+        resp = func(*args, **kwargs)
+        if resp['code'] == 200 and resp['data_type'] == 'table':
+            data_rows = resp['data']
+            if sort:
+                heading = data_rows[0]
+                data_rows = data_rows[1:]
+                index = -1
+                if sort.upper() in heading:
+                    index = heading.index(sort.upper())
```

Comment:
Unhandled exception - will raise a `ValueError`.

Reply:
Please look at [this](https://stackoverflow.com/a/2294502/5312750)

Reply:
sir if the value is not present ....that line itself is not being executed...still i will change it find

Reply:
yeah, you're right. `index` is fine. My mistake.